### ISSUE TITLE
Profiler

### ISSUE THREAD

**Issue by tknopp** [tknopp/Julietta.jl#4]
could be integrated using a tree view

**Comment by timholy** [comment]
Right, although `ProfileView.jl` is even nicer. I can port to Gtk sometime.

**Comment by tknopp** [comment]
Wow I was not aware of that! Looks nice! This stub issue was opened for exactly this reason!

**Comment by RoyiAvital** [comment]
Is the project still alive?

**Comment by tknopp** [comment]
No currently not
